Assets/Scripts/Models/DialogBoxInformation/DialogComponent.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Serialization;
+using UnityEngine;
+
+public class DialogComponent
+{
+    [XmlAttribute("Type")]
+    public string ObjectType;
+
+    [XmlElement]
+    public Vector3 position;
+
+    [XmlElement]
```

Comment:
ah ye, sorry.. copy-paste failed me, just wanted to put there capitals
[XmlElement("Data")]